exec/java-exec/src/main/java/org/apache/drill/exec/work/metadata/ServerMetaProvider.java

```diff
@@ -76,7 +76,7 @@
       .setReadOnly(false)
       .setGroupBySupport(GroupBySupport.GB_UNRELATED)
       .setLikeEscapeClauseSupported(true)
-      .setNullCollation(NullCollation.NC_AT_END)
+      .setNullCollation(NullCollation.NC_HIGH)
```

Comment:
I'm not completely sure why we should change from NC_AT_END to NC_HIGH, in stead of NC_AT_END.  I thought Drill is using ASC as default ordering, and NULLS LAST as default null collation for ASC. This is consistent to what Oracle [1] and Postgres [2] is doing : ASC /NULL LAST is the default option.  

1. http://docs.oracle.com/javadb/10.6.2.1/ref/rrefsqlj13658.html
2. https://www.postgresql.org/docs/9.4/static/queries-order.html